src/test/java/specs/ParameterizedExampleSpecs.java

```diff
@@ -118,6 +124,24 @@
             example("bye", -1.5))
 
     );
+
+    Supplier<Set<String>> set = let(HashSet::new);
+    scenarioOutline("behaviour of let inside scenario outline",
+        (first, second) -> {
+          given("set contains first", () -> {
+            set.get().add(first);
+          });
+          when("set has second added", () -> {
+            set.get().add(second);
+          });
+          then("both are found in the set", () -> {
+            assertTrue(set.get().contains(first));
+            assertTrue(set.get().contains(second));
```

Comment:
Even after one of the fixes, this wasn't going to work because the `scenarioOutline` wasn't generating actual scenarios...